src/pages/documentation/release-notes/index.md

```diff
@@ -4,6 +4,14 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## October 27, 2022
+
+### iOS AEPCore 3.7.2
```

Comment:
@cdhoffmann let's also add it under mobile core release notes https://developer-stage.adobe.com/client-sdks/documentation/mobile-core/release-notes/

Reply:
good call, i always forget.